### PR TITLE
1.21 Support

### PR DESCRIPTION
**Pull request by JustinDevB**
Added support for 1.21

### CODE REVIEW

File: src/main/java/me/jumper251/replay/utils/VersionUtil.java [hunk 1]
```diff
@@ -67,7 +67,10 @@ public enum VersionEnum {
 		V1_17(10),
 		V1_18(11),
 		V1_19(12),
-		V1_20(13);
+		V1_20(13),
+		CRAFTBUK(14),
```
Comment (Jumper251) [inline src/main/java/me/jumper251/replay/utils/VersionUtil.java:71]: Is this used anywhere?

Comment (JustinDevB) [reply src/main/java/me/jumper251/replay/utils/VersionUtil.java:71]: Paper remaps plugins at runtime now. If "CRAFTBUK" isn't in there the plugin throws errors

Comment (JustinDevB) [reply src/main/java/me/jumper251/replay/utils/VersionUtil.java:71]: java.lang.IllegalArgumentException: No enum constant me.jumper251.replay.utils.VersionUtil.VersionEnum.CRAFTBUK
        at java.base/java.lang.Enum.valueOf(Enum.java:293) ~[?:?]
        at org.bukkit.craftbukkit.legacy.FieldRename.valueOf(FieldRename.java:54) ~[purpur-1.20.6.jar:1.20.6-2230-eac4198]
        at Replay.jar/me.jumper251.replay.utils.VersionUtil$VersionEnum.BUKKIT_CUSTOM_METHOD_org_bukkit_craftbukkit_legacy_FieldRename_valueOf(VersionUtil.java) ~[Replay.jar:?]
        at Replay.jar/me.jumper251.replay.utils.VersionUtil$VersionEnum.valueOf(VersionUtil.java:56) ~[Replay.jar:?]
        at Replay.jar/me.jumper251.replay.utils.VersionUtil.isBelow(VersionUtil.java:31) ~[Replay.jar:?]
        at Replay.jar/me.jumper251.replay.replaysystem.recording.PacketRecorder.register(PacketRecorder.java:101) ~[Replay.jar:?]
        at Replay.jar/me.jumper251.replay.replaysystem.recording.Recorder.start(Recorder.java:69) ~[Replay.jar:?]
        at Replay.jar/me.jumper251.replay.replaysystem.Replay.recordAll(Replay.java:51) ~[Replay.jar:?]
        at Replay.jar/me.jumper251.replay.api.ReplayAPI.recordReplay(ReplayAPI.java:55) ~[Replay.jar:?]
        at Replay.jar/me.jumper251.replay.api.ReplayAPI.recordReplay(ReplayAPI.java:49) ~[Replay.jar:?]
        at AntiCheatReplay-2.7.12.jar/me.justindevb.anticheatreplay.ListenerBase.lambda$startRecording$1(ListenerBase.java:85) ~[AntiCheatReplay-2.7.12.jar:?]
        at org.bukkit.craftbukkit.scheduler.CraftTask.run(CraftTask.java:101) ~[purpur-1.20.6.jar:1.20.6-2230-eac4198]
        at org.bukkit.craftbukkit.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:482) ~[purpur-1.20.6.jar:1.20.6-2230-eac4198]
        at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1734) ~[purpur-1.20.6.jar:1.20.6-2230-eac4198]
        at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:503) ~[purpur-1.20.6.jar:1.20.6-2230-eac4198]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1606) ~[purpur-1.20.6.jar:1.20.6-2230-eac4198]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1260) ~[purpur-1.20.6.jar:1.20.6-2230-eac4198]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:326) ~[purpur-1.20.6.jar:1.20.6-2230-eac4198]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]

**Comment by Jumper251** [comment]
Most things seem to work but because of the changes with Paper the version is detected as craftbuk.
This breaks some stuff so I have to update the Version detection before I release the update.

**Comment by JustinDevB** [comment]
Yeah, it wasn’t a pretty fix. But it worked enough to get the plugin functioning so my plugin that depends on AdvancedReplay could get updated.

**Pull request merged**